src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -887,6 +881,35 @@ LogicalResult ONNXSeluOp::inferShapes(
 
 LogicalResult ONNXSequenceInsertOp::inferShapes(
```

Comment:
Did you add verifiers to make sure the elementary types are identical?

Reply:
Yes. Verifier was added in a previous PR.